File: supporting docs/Uncolored (Risks Assesment Tool)/src/uncolored/controllers/DashboardController.java
package uncolored.controllers;

import java.io.IOException;
import java.net.URL;
import java.util.ResourceBundle;
import java.util.logging.Level;
import java.util.logging.Logger;
import javafx.fxml.FXML;
import javafx.fxml.FXMLLoader;
import javafx.fxml.Initializable;
import javafx.scene.Node;
import javafx.scene.input.MouseEvent;
import javafx.scene.layout.HBox;
import javafx.scene.layout.VBox;

/**
 *
 * @author Benny
 */
public class DashboardController implements Initializable {

    @FXML
    private VBox content;

    VBox current, newAnalysis, savedAnalysis, baselineAnalysis, setting;

    @Override
    public void initialize(URL url, ResourceBundle rb) {
        try {
            current = FXMLLoader.load(getClass().getResource("/uncolored/views/Current.fxml"));
            newAnalysis = FXMLLoader.load(getClass().getResource("/uncolored/views/NewAnalysis.fxml"));
            savedAnalysis = FXMLLoader.load(getClass().getResource("/uncolored/views/SavedAnalysis.fxml"));
            baselineAnalysis = FXMLLoader.load(getClass().getResource("/uncolored/views/BaselineAnalysis.fxml"));
            setting = FXMLLoader.load(getClass().getResource("/uncolored/views/Setting.fxml"));
            content.getChildren().setAll(current);

        } catch (IOException ex) {
            Logger.getLogger(DashboardController.class.getName()).log(Level.SEVERE, null, ex);
        }
    }

    @FXML
    private void changeContent(MouseEvent event) {
        content.getChildren().removeAll();

        HBox btn = (HBox) ((Node) event.getSource());
        switch (btn.getId()) {
            case "currentRisk":
                content.getChildren().setAll(current);
                break;
            case "nAnalysis":
                content.getChildren().setAll(newAnalysis);
                break;
            case "sAnalysis":
                content.getChildren().setAll(savedAnalysis);
                break;
            case "bAnalysis":
                content.getChildren().setAll(baselineAnalysis);
                break;
            case "setting":
                content.getChildren().setAll(setting);
                break;
        }
    }

}

File: supporting docs/UnColored .NET/src/uncolored/controllers/DashboardController.cs
namespace uncolored.controllers
{
	using FXML = javafx.fxml.FXML;
	using FXMLLoader = javafx.fxml.FXMLLoader;
	using Initializable = javafx.fxml.Initializable;
	using Node = javafx.scene.Node;
	using MouseEvent = javafx.scene.input.MouseEvent;
	using HBox = javafx.scene.layout.HBox;
	using VBox = javafx.scene.layout.VBox;

	/// 
	/// <summary>
	/// @author Benny
	/// </summary>
	public class DashboardController : Initializable
	{

//JAVA TO C# CONVERTER TODO TASK: Most Java annotations will not have direct .NET equivalent attributes:
//ORIGINAL LINE: @FXML private javafx.scene.layout.VBox content;
		private VBox content;

		internal VBox current, newAnalysis, savedAnalysis, baselineAnalysis, setting;

		public override void initialize(URL url, ResourceBundle rb)
		{
			try
			{
				current = FXMLLoader.load(this.GetType().getResource("/uncolored/views/Current.fxml"));
				newAnalysis = FXMLLoader.load(this.GetType().getResource("/uncolored/views/NewAnalysis.fxml"));
				savedAnalysis = FXMLLoader.load(this.GetType().getResource("/uncolored/views/SavedAnalysis.fxml"));
				baselineAnalysis = FXMLLoader.load(this.GetType().getResource("/uncolored/views/BaselineAnalysis.fxml"));
				setting = FXMLLoader.load(this.GetType().getResource("/uncolored/views/Setting.fxml"));
				content.Children.All = current;

			}
			catch (IOException ex)
			{
//JAVA TO C# CONVERTER WARNING: The .NET Type.FullName property will not always yield results identical to the Java Class.getName method:
				Logger.getLogger(typeof(DashboardController).FullName).log(Level.SEVERE, null, ex);
			}
		}

//JAVA TO C# CONVERTER TODO TASK: Most Java annotations will not have direct .NET equivalent attributes:
//ORIGINAL LINE: @FXML private void changeContent(javafx.scene.input.MouseEvent event)
		private void changeContent(MouseEvent @event)
		{
			content.Children.removeAll();

			HBox btn = (HBox)((Node) @event.Source);
			switch (btn.Id)
			{
				case "currentRisk":
					content.Children.All = current;
					break;
				case "nAnalysis":
					content.Children.All = newAnalysis;
					break;
				case "sAnalysis":
					content.Children.All = savedAnalysis;
					break;
				case "bAnalysis":
					content.Children.All = baselineAnalysis;
					break;
				case "setting":
					content.Children.All = setting;
					break;
			}
		}

	}

}